File: 프로그래머스/1/131705. 삼총사/삼총사.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> number) {
    int r=0,l=number.size();
    for(int i=0;i<l-2;i++) {
        for(int j=i+1;j<l-1;j++) {
            for(int k=j+1;k<l;k++) {
                if(number[i]+number[j]+number[k]==0) {
                    r+=1;
                }
            }
        }
    }
    return r;
}

File: 프로그래머스/1/131705. 삼총사/삼총사.java
class Solution {
    public int solution(int[] number) {
        int rlt = 0,len=number.length;
        for(int i=0;i<len-2;i++) {
            for(int j=i+1;j<len-1;j++) {
                for(int k=j+1;k<len;k++) {
                    if(number[i]+number[j]+number[k]==0)
                        rlt+=1;
                }
            }
        }
        return rlt;
    }
}